app/src/main/res/values/fonts.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="default_typeface_regular">fonts/Quicksand-Regular.ttf</string>
+  <string name="default_typeface_medium">fonts/Quicksand-Medium.ttf</string>
+  <string name="default_typeface_light">fonts/Quicksand-Light.ttf</string>
+  <string name="default_typeface_bold">fonts/Quicksand-Bold.ttf</string>
+  <string name="title_typeface">fonts/leaguespartan-bold.ttf</string>
```

Comment:
This was supposed to be called `leaguespartan-bold.otf`, I think you got the wrong version of the font